src/components/CounterAccordion.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from './Icon';
+
+class CounterAccordion extends Component {
+  state = { collapsed: true };
+
+  handleClick = () => {
+    this.setState(state => {
+      return {
+        collapsed: !state.collapsed,
+      };
+    });
+  };
+
+  render() {
+    const { icon, label, minHeight, count, children } = this.props;
+    const { collapsed } = this.state;
+    const accordionClasses = classnames('accordion-label', { collapsed });
+    const panelHeight = collapsed ? minHeight : (count + 1) * minHeight;
+
+    return (
+      <li
+        className={accordionClasses}
+        style={{ maxHeight: panelHeight }}
+        onClick={this.handleClick}
+      >
+        <a>
```

Comment:
We can make this `div` now or remove completely.

Reply:
I agree.. but I stuck with `li > a` structure in order to avoid adding more rules to `sidebar.scss`.
In the present form, the elements will simply use existing styles..

Whatsay?